bundles/org.openhab.binding.buienradar/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -50,9 +51,28 @@
 				<description>Refresh interval in minutes</description>
 				<default>5</default>
 			</parameter>
+			
+			<parameter name="retries" type="integer" required="true" min="1" >
+                <label>Retries</label>
+                <description>Number of retries to try to retrieve buienradar results</description>
+                <default>4</default>
+            </parameter>
+			
+			<parameter name="exponentialBackoffRetryBaseInSeconds" type="integer" required="true" min="1" unit="s">
```

Comment:
Please check the formatting in this file

Reply:
Ah, I thought teh xml files would also auto-format on save...